src/components/SlateEditor/plugins/concept/SlateConceptPreview.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { DeleteForever } from '@ndla/icons/editor';
+import { Link as LinkIcon } from '@ndla/icons/common';
+import { injectT, tType } from '@ndla/i18n';
+import {
+  NotionDialogContent,
+  NotionDialogText,
+  NotionDialogLicenses,
+} from '@ndla/notion';
+import Tooltip from '@ndla/tooltip';
+import { addShowConceptDefinitionClickListeners } from '@ndla/article-scripts';
+import { Concept as ConceptType } from '../../editorTypes';
+import IconButton from '../../../IconButton';
+
+const StyledFigureButtons = styled('div')`
+  position: absolute;
+  top: 0;
+  z-index: 1;
+  right: -${spacing.spacingUnit * 1.5}px;
+  margin-right: 40px;
```

Comment:
Vil egentlig ikke hardkode denne inn, men den Headeren i Notion komponenten var ikke veldig lett å endre på uten å endre på komponenten. large ble for mye og medium spacing ble for lite.